### PR TITLE
feat(runbotics-orchestrator-ui): :sparkles: [RPA-1431] Adding the "RunBotics Dream Team" section to the landing page

### PR DESCRIPTION
**Pull request by AdriannaSuchomska**
## Description
Adding the "RunBotics Dream Team" section to the landing page

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

---

I agree to the terms of the RunBotics Contributor License Agreement.

### CODE REVIEW

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/AboutTeamSection/AboutTeamNumbers/AboutTeamNumbersItem.module.scss [hunk 1]
```diff
@@ -0,0 +1,16 @@
+@import '#src-landing/scss/main';
+
+.number{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    height: 80px;
+    max-width: 30%;
+    padding-left: $padding-1;
+    border-left: 4px solid #FFFFFF;
```
Comment (maciejKrasny) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/AboutTeamSection/AboutTeamNumbers/AboutTeamNumbersItem.module.scss]: Please use color from variable

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/AboutTeamSection/AboutTeamSection.module.scss [hunk 2]
```diff
@@ -0,0 +1,176 @@
+@import '#src-landing/scss/main';
+
+
+.background{
+    width: 100%;
+    background: linear-gradient(to bottom, white 65%, #F9B934 65%);
```
Comment (maciejKrasny) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/AboutTeamSection/AboutTeamSection.module.scss]: Please use color from variable

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/AboutTeamSection/AboutTeamSection.module.scss [hunk 3]
```diff
@@ -0,0 +1,176 @@
...
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    width: 70%;
+}
+
+.manager img {
+    margin-right: $margin-3;
+    padding: 5px;
+    border: 1px solid #000000;
```
Comment (maciejKrasny) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/AboutTeamSection/AboutTeamSection.module.scss]: Please use color variable

File: runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/AboutTeamSection/AboutTeamSection.module.scss [hunk 4]
```diff
@@ -0,0 +1,176 @@
...
+    grid-row: span 2;
+    height: 786px;
+    margin: auto;
+    margin-left: 0;
+    margin-right: $margin-3;
+    border-radius: 20px 20px 150px 20px;
+    object-fit: cover;
+}
+
+.numbers{
```
Comment (maciejKrasny) [inline runbotics/runbotics-orchestrator-ui/src/src-landing/views/sections/AboutTeamSection/AboutTeamSection.module.scss]: File format

**Review by maciejKrasny — APPROVED** [review]

File: runbotics/runbotics-orchestrator-ui/public/images/photos/team.jpg
Comment (marcin-depcik) [inline runbotics/runbotics-orchestrator-ui/public/images/photos/team.jpg:1]: <3

**Review by marcin-depcik — APPROVED** [review]
Bump version

**Pull request merged**